### PR TITLE
Doc update: added notes on endpoint gateway reserved ip

### PR DESCRIPTION
**Pull request by deepaksibm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown [hunk 1]
```diff
@@ -73,6 +73,9 @@ The following arguments are supported:
 - `resource_group` - (Optional, string,ForceNew)The resource group id
 - `tags` - (Optional, array of strings) Tags associated with the instance.
 
+**NOTE**: For `ips` , do not provide `id` and `subnet` together. Provider either one of them per configuration block.
+**NOTE**: `ips` once configured inline in this resource are not modifiable.  Prefer using `ibm_is_virtual_endpoint_gateway_ip` resource to bind/unbind new reserved IPs to endpointgateways and use the resource `ibm_is_subnet_reserved_ip` to create new reserved IPs
```
Comment (astha-jain) [inline website/docs/r/is_virtual_endpoint_gateway.html.markdown:77]: **NOTE**: `id` and `subnet` are mutually exclusive. 
And please move this under `ips` block

Comment (astha-jain) [reply website/docs/r/is_virtual_endpoint_gateway.html.markdown:77]:
```
**NOTE**: `ips` configured inline in this resource are not modifiable. Prefer using `ibm_is_virtual_endpoint_gateway_ip` resource to bind/unbind new reserved IPs to endpoint gateways and use the resource `ibm_is_subnet_reserved_ip` to create new reserved IP. 
```
Made some changes to the block

**Review by astha-jain — APPROVED** [review]
Please resolve conflicts and address comments. 
This PR with the suggested changes is good to go.

**Comment by deepaksibm** [comment]
Created PR to upstream.
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/2761

**Pull request closed without merging**